com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md

```diff
@@ -2,13 +2,24 @@
 
 The following process uses DaVinci Resolve, but you can apply it for other digital content creation tools such as [Photoshop](LUT-Authoring-Photoshop.md) or Nuke.
 
-## Step 1: Set up the HDRP Asset
+To author lookup textures in resolve: 
+
+1. [Set up the HDRP Asset](#set-up-hdrp-asset)
```

Comment:
I think each of these bullet items needs a full stop at the end.

Reply:
Good point, thanks!